generators/chipyard/src/main/scala/ChipTop.scala

```diff
@@ -31,5 +31,5 @@ class ChipTop(implicit p: Parameters) extends LazyModule with BindingScope
   // of ChipTop (ex: ClockGroup) do not receive clock or reset.
   // However. anonymous children of ChipTop should not need an implicit Clock or Reset
   // anyways, they probably need to be explicitly clocked.
-  lazy val module: LazyModuleImpLike = new LazyRawModuleImp(this) {  }
+  lazy val module: LazyModuleImpLike = new LazyRawModuleImp(this) with DontTouch {  }
```

Comment:
Why is this needed?

Reply:
Previously, we used IOBinders to dontTouch ChipTop ports. 

Now, we use this rocketchip util. This way, the dontTouch will work correctly on designs without iobinders.